docs/rst/notes/forthcoming_version.rst

```diff
@@ -11,6 +11,9 @@ This release includes the following **mayor changes**:
   - `ddsrouter_core`: C++ library which implements the DDS Router operation and exports the DDS Router C++ API.
   - `ddsrouter_yaml`:  C++ library to parse the DDS Router *yaml* configuration files.
   - `ddsrouter_tool`: DDS Router end-user application.
+* New dynamic topic discovery feature: the user does not need to fully specify over which topics to communicate (i.e.
+  provide concrete topic names and types). The discovery of topics matching the allowlisting rules automatically
+  triggers the creation of all entities required for communication. See :ref:`topic_filtering` for more details.
```

Comment:
* Build internal topic tracks via the `builtin-topics` configuration.

This release includes the following **Routing features**:

* Apply DDS Topic filter rules using the `allowlist` and `blocklist` lists.